src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2988,18 +2988,20 @@ because xbuild doesn't support framework reference assemblies.
 <Target Name="_DeployAppBundle"
     Condition=" '$(AndroidPackageFormat)' == 'aab' "
     DependsOnTargets="_BuildApkSet">
+  <!-- The previous attempt failed. So yry to uninstall but save the cache using cmd package uninstall -k -->
   <AndroidAdb
       Condition=" '$(EmbedAssembliesIntoApk)' == 'true' "
       ContinueOnError="True"
       ToolExe="$(AdbToolExe)"
       ToolPath="$(AdbToolPath)"
       AdbTarget="$(AdbTarget)"
-      Command="uninstall"
-      Arguments="-k $(_AndroidPackage)"
+      Command="shell cmd package uninstall -k"
```

Comment:
I wonder if this should instead be `adb shell pm uninstall -k PACKAGE`.

Reply:
…as I'd never heard of `adb shell cmd` before now, and it's not listed at e.g. <adbshell.com>.

Reply:
I didn't notice it's doing `adb shell cmd package uninstall -k`.

android-platform-support looked like it was doing `adb shell pm uninstall -k`, so let me just change to that and let's see what CI says?